src/app/register/register.component.html

```diff
@@ -83,6 +83,15 @@
         </div>
       </form>
       <div class="actions">
+        @if (flags.friendlyCaptchaEnabled) {
+          <!-- Sitekey below is shared between environments and necessarily exposed in FE, so should be fine to hard-code -->
+          <div
+            class="frc-captcha"
+            data-sitekey="FCMIOJ2ARSHLBGAJ"
```

Comment:
I think the key is intended to be shared between envs - I couldn't actually find anything in the [friendly captcha docs](https://docs.friendlycaptcha.com/#/)  to confirm that or say if we should be setting each env up as a separate site.